src/booleans.js

```diff
@@ -1,61 +1,135 @@
 const negate = a => {
-  // your code here
+  if (a) {
+	return false
+}
+  else {
+	return true
+}		
 };
 
 const both = (a, b) => {
-  // your code here
+  if (a == true && b == true) {
+	return true
+}
+  else{
+	return false
+}
 };
 
 const either = (a, b) => {
-  // your code here
+  if (a == true || b == true) {
+	return true
+}
+  else {
+	return false
+}
+
 };
 
 const none = (a, b) => {
-  // your code here
+  if(a !== true && b !== true) {
+	return true
+}
+  else {
+	return false
+}
 };
 
 const one = (a, b) => {
-  // your code here
+  if (a == true && b !== true) {
+	return true
+}
+ else if (a !== true && b == true) {
+	return true
+}
+ else {
+	return false
+}
+
 };
 
 const truthiness = a => {
-  // your code here
+  if (a) {
+	return true
+}
+  else {
+	return false
+}
+
 };
 
 const isEqual = (a, b) => {
-  // your code here
+  if (a == b) {
+	return true
+}
+  else{
+	return false
+}
 };
 
 const isGreaterThan = (a, b) => {
-  // your code here
+  if (a > b) {
+	return true
+}
+  else {
+	return false
+}
 };
 
 const isLessThanOrEqualTo = (a, b) => {
-  // your code here
+  if (a <= b) {
+	return true
+}
+else {
+	return false
+}
 };
 
 const isOdd = a => {
-  // your code here
+  if (Number.isInteger(a / 2) == false) {
+	return true
+}
+  else {
+	return false
+}
 };
 
 const isEven = a => {
-  // your code here
+  return Number.isInteger(a / 2)
 };
 
 const isSquare = a => {
-  // your code here
+  return Number.isInteger(Math.sqrt(a))
 };
 
 const startsWith = (char, string) => {
-  // your code here
+  if (string[0] === char) {
+	return true
+}
+  else {
+	return false
+}
 };
 
 const containsVowels = string => {
-  // your code here
+   for (let i = 0; i <= string.length; i++){
+
+       if(string.includes('a') || string.includes('A') || string.includes('e') || string.includes('E') || string.includes('I') || string.includes('i') || string.includes('o') || string.includes('O') || string.includes('u') || string.includes('U')) {
```

Comment:
You don't need the for loop here.

You can start by ensuring that the string is lowercase with:
const lower = string.toLowerCase();

And then you can do:
return (lower.includes('a') || lower.includes('e') || and it goes on with all of the vowels)